.ci/.grenrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    "username": "elastic",
+    "repo": "apm-agent-php",
+    "dataSource": "prs",
+    "groupBy": {
+        "Breaking changes": ["breaking", "automation"],
+        "Bug Fixes": ["bug", "fix"],
+        "Features": ["enhancement", "internal", "feature", "feat", "documentation"]
```

Comment:
NIT: for testing purposes

```suggestion
        "Features": ["enhancement", "internal", "feature", "feat"]
```